projects/Inventaire/module.nix

```diff
@@ -20,7 +20,7 @@ let
   settingsLinks = ''
     Links for settings (make sure to look at the version that's actually packaged):
     - <a href="https://codeberg.org/inventaire/inventaire/src/branch/main/server/types/config.ts">Types</a>
-    - <a href="https://codeberg.org/inventaire/inventaire/src/branch/main/config/default.cjs">Defaults</a>
+    - <a href="https://codeberg.org/inventaire/inventaire/src/branch/main/config/default.ts">Defaults</a>
```

Comment:
Same suggestion as https://github.com/ngi-nix/ngipkgs/pull/1734/files#r2480377771

Reply:
If that's the case, we shouldn't change this, since the [current inventaire version's default config file](https://codeberg.org/inventaire/inventaire/src/tag/v4.0.1/config/default.cjs) has the `.cjs` extension.

Reply:
Reverted to cjs for this file. I will squash all commits once you review.